tests/run_tests.py

```diff
@@ -34,6 +34,10 @@
 print "Adding tests/python location to python_path: %s" % test_python_path
 sys.path = [test_python_path] + sys.path
 
+test_python_path = os.path.join(test_python_path, "3rdParty")
```

Comment:
i guess we should remove these:

- https://github.com/shotgunsoftware/tk-core/tree/master/tests/python/unittest2
- https://github.com/shotgunsoftware/tk-core/blob/master/tests/python/mock.py